front/components/assistant/conversation/WebsearchAction.tsx

```diff
@@ -25,6 +26,19 @@ export default function WebsearchAction({
 
   const { results } = websearchAction.output ?? { results: [] };
 
+  const rawError =
+    websearchAction.output && "error" in websearchAction.output
```

Comment:
NIT: shouldn't you update `WebsearchActionOutputType` to set that there's an optional error string so this code could be simplified?

Reply:
can't make an optional field with io-ts unfortunately AFAIK (after having tried)